pkg/service/controller/servingcert/secret_creating_controller.go

```diff
@@ -162,7 +161,7 @@ func (sc *ServiceServingCertController) enqueueService(obj interface{}) {
 	if !ok {
 		return
 	}
-	key, err := controller.KeyFunc(obj)
+	key, err := cache.DeletionHandlingMetaNamespaceKeyFunc(obj)
```

Comment:
nit: the cast to `v1.Service` above will prevent check of `obj == DeletedFinalStateUnknown` right? So you can just use meta.MetaNamespaceKeyFunc

Reply:
> nit: the cast to v1.Service above will prevent check of obj == DeletedFinalStateUnknown right? So you can just use meta.MetaNamespaceKeyFunc

This one is the fully portable version.  I'd like to keep using everywhere we can.

Reply:
k, i guess we also want to fix the image import scheduler https://github.com/openshift/origin/pull/19687